workflows/devnet-build/Jenkinsfile

```diff
@@ -42,11 +46,14 @@ pipeline{
         }
         stage('Upload tarball'){
             steps {
-                dir("lisk-build") {
-                    sh """
-                    s3cmd put --acl-public release/lisk-${env.VERSION}-Linux-x86_64.tar.gz  s3:///lisk-releases/lisk-core/${params.tag}/lisk-${env.VERSION}-Linux-x86_64.tar.gz
-                    s3cmd put --acl-public release/lisk-${env.VERSION}-Linux-x86_64.tar.gz.SHA256 s3:///lisk-releases/lisk-core/${params.tag}/lisk-${env.VERSION}-Linux-x86_64.tar.gz.SHA256
-                    """
+                script {
+                    def outputFile = params.commitsha ? "lisk-${env.VERSION}-${env.COMMIT}-Linux-x86_64.tar.gz" : "lisk-${env.VERSION}-Linux-x86_64.tar.gz"
```

Comment:
Won't the `*.SHA256` contain the wrong file name?

Reply:
Yes you're right here... I will need to re-generate it